### PR TITLE
zh-cn for Tour: multiple parameter lists

### PR DESCRIPTION
**Pull request by declan94**
Simplified Chiniese version for Tour: multiple parameter lists

### CODE REVIEW

File: _zh-cn/tour/multiple-parameter-lists.md [hunk 1]
```diff
@@ -13,3 +13,71 @@ language: zh-cn
 next-page: case-classes
 previous-page: nested-functions
 ---
+
+方法可以定义多个参数列表，当使用较少的参数列表调用多参数列表的方法时，会产生一个新的函数，该函数接收剩余的参数列表作为其参数。这被称为[柯里化](https://zh.wikipedia.org/wiki/%E6%9F%AF%E9%87%8C%E5%8C%96)。
+
+下面是一个例子，在Scala集合中定义的特质[Traversable](/overviews/collections/trait-traversable.html)：
```
[the remark this answers is not in the document]
Comment (declan94) [reply _zh-cn/tour/multiple-parameter-lists.md]: You mean the link jumps to the English version?

[the remark this answers is not in the document]
Comment (declan94) [reply _zh-cn/tour/multiple-parameter-lists.md]: Changed the link in the newest commit and now it points to the Chinese version

[the remark this answers is not in the document]
Comment (declan94) [reply _zh-cn/tour/multiple-parameter-lists.md]: For me It works. Have you followed the following instructions in the README?

## Quickstart ##

To build and view the site locally:

    gem install bundler
    bundle install
    bundle exec jekyll serve -I

File: _zh-cn/tour/multiple-parameter-lists.md [hunk 2]
```diff
@@ -13,3 +13,71 @@ language: zh-cn
...
+```
+numbers.foldLeft(0, {(m: Int, n: Int) => m + n})
+```
+   
+   注意使用多参数列表时，我们还可以利用Scala的类型推断来让代码更加简洁（如下所示），而如果没有多参数列表，这是不可能的。
+    
+```
+numbers.foldLeft(0)(_ + _)
+```
+   上述声明`numbers.foldLeft(0)(_ + _)`让我们可以指定参数`z`，然后再传递折叠函数，并且可以复用，如下所示：
```
Comment (dongxuwang) [inline _zh-cn/tour/multiple-parameter-lists.md]:
```
-让我们可以指定参数`z`，然后再传递折叠函数，并且可以复用，如下所示：
+上述语句让我们可以固定参数`z`来形成一个部分函数进行传递，达到复用的目的，如下所示：
```

Comment (dongxuwang) [reply _zh-cn/tour/multiple-parameter-lists.md]: After doing some research<sup>[1]</sup><sup>[2]</sup>, I kind of believe that's what's supposed to mean, but not 100 percent sure.
<sup>1</sup> https://www.grokacademy.com/lessons/demystifying-currying-and-partial-function-application/
<sup>2</sup> https://goo.gl/oGhpTW

Comment (Qi77Qi) [reply _zh-cn/tour/multiple-parameter-lists.md]: I think all it wants to say is `currying allows you to fix z and fold the rest of the collection with the partially applied function`... `Above statement numbers.foldLeft(0)(_ + _)` in the Eng version is a bit confusing

Comment (declan94) [reply _zh-cn/tour/multiple-parameter-lists.md]: @dongxuwang I think you're right.

Comment (dongxuwang) [reply _zh-cn/tour/multiple-parameter-lists.md]: @Qi77Qi Agree!
It always makes us crazy that partially applied function, partially function application, partial function. together with the PartialFunction etc.

**Comment by SethTisue** [comment]
thank you!

**Pull request merged**